src/writer/titleListingHandler.h

```diff
@@ -36,20 +36,18 @@ struct TitleCompare {
 
 class TitleListingHandler : public DirentHandler {
   public:
-    typedef std::vector<Dirent*> Dirents;
-
     explicit TitleListingHandler(CreatorData* data);
     virtual ~TitleListingHandler();
 
     void start() override;
     void stop() override;
     bool isCompressible() override { return false; }
-    std::unique_ptr<ContentProvider> getContentProvider() const override;
+    ContentProviders getContentProviders() const override;
     void handle(Dirent* dirent, std::shared_ptr<Item> item) override;
     void handle(Dirent* dirent, const Hints& hints) override;
 
   protected:
-    Dirent* createDirent() const override;
+    Dirents createDirents() const override;
     CreatorData* mp_creatorData;
     Dirents m_dirents;
```

Comment:
`Dirents createDirents()` and `Dirents m_dirents` can be a source of serious confusion. It should be noted that `m_dirents` is **not** the result of `createDirents()` and its meaning must be clarified.

Reply:
`m_dirents` is now renamed to `m_handledDirents`.